_specs/ecip-1102.md

```diff
@@ -0,0 +1,139 @@
+---
+ecip: 1102
+title: Use The Etherplan Consensus Engine for Approving Proto Treasury Funding Proposals
+lang: EN
+author: Donald McIntyre (@TokenHash)
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/407
+status: WIP
+type: Meta
+created: 2021-02-03
+complementary to: ECIP-1098 Proto Treasury System 
```

Comment:
I did not see that ECIP-1000 does not permit other header fields not specified, but yes, I invented `complementary to` to indicate that this ECIP only works if the other ECIP is implemented. It can be deleted if you prefer.